src/data_storage/graph/node_filter.ts

```diff
@@ -0,0 +1,19 @@
+import Filter from "../filter";
+import NodeStorage from "./node_storage";
+import {NodeT} from "../../types/graph/nodeT";
+import Result from "../../result";
+
+export default class NodeFilter extends Filter {
+    constructor() {
+        super(NodeStorage.tableName);
+    }
+
+    metatypeID(operator: string, value: any) {
+        super.query("metatype_id", operator, value)
```

Comment:
Is operator defined?

Reply:
Not sure what you mean. Operator will be the supported list of operators like `eq` and `neq`

Reply:
Oh I just totally misread the metatypeID parameters. Looks good.